verification/curator-service/ui/src/components/FeedbackEmailDialog/index.tsx

```diff
@@ -25,13 +25,27 @@ export interface FeedbackFormValues {
 
 const FeedbackEmailDialog = ({
     isOpen,
-    handleClose,
+    closeFeedbackModal,
 }: FeedbackEmailDialogProps): JSX.Element => {
-    const classes = useStyles();
     const user = useAppSelector(selectUser);
 
     const maxSizeMessage = 800;
 
+    const [alertInformationOpen, setAlertInformationOpen] =
+        useState<boolean>(false);
+    const [alertInformationMessage, setAlertInformationMessage] =
+        useState<string>('');
+    const [alertInformationType, setAlertInformationType] = useState<
```

Comment:
In cases like this you can also add all those state variables under one state object. This way you later have to only call `setState` once providing all the values instead of three times